app/controllers/event_steps_controller.rb

```diff
@@ -1,10 +1,15 @@
 class EventStepsController < ApplicationController
   before_action :load_event
-  before_action :redirect_closed_events, only: %i[show update] # rubocop:disable Rails/LexicallyScopedActionFilter
 
   include WizardSteps
   self.wizard_class = Events::Wizard
 
+  # rubocop:disable Rails/LexicallyScopedActionFilter
```

Comment:
I think its probably better to exclude these files from that rule in rubocops config? Than there's just one place to add any future wizard controllers. Could potentially event be a wildcard on `app/controllers/**/*steps_controller.rb` ?